documentation/WiFi_Direct_Support.md

```diff
@@ -0,0 +1,18 @@
+## Wi-Fi Direct Support
+
+With Wi-Fi Direct, the Physical Web will allow beacons to broadcast and serve data to users. The advantage this provides is allowing devices without internet connectivity to access data. 
```

Comment:
start off by giving a brief intro to wifi direct, also link to some resource that explains it further.